### ISSUE TITLE
where is class-balanced cross-entropy loss in the code

### ISSUE THREAD

**Issue by Lzc6996** [s9xie/hed#38]
In your paper, there is  "we define the following class-balanced cross-entropy
loss function used in Equation", but i can't find it in your code. In the SigmoidCrossEntropyLoss , it is just a common CrossEntropyLoss without class-balanced .
